Makefile

```diff
@@ -1,9 +1,9 @@
 GO ?= $(shell command -v go 2> /dev/null)
-DEP ?= $(shell command -v dep 2> /dev/null)
 NPM ?= $(shell command -v npm 2> /dev/null)
 HTTP ?= $(shell command -v http 2> /dev/null)
 CURL ?= $(shell command -v curl 2> /dev/null)
 MANIFEST_FILE ?= plugin.json
+export GO111MODULE=on
```

Comment:
I think this should be optional by current working directory. I know many people dislike GOPATH, but I like to use GOPATH and GOPATH-like workspace even in other languages.

Reply:
Having `GO111MODULE` set to `on` should not prevent people from placing the code in `GOPATH`, does it?

Reply:
oh.... you right, please never mind. I can't remember why I thought it would prevent to use GOPATH :sob:

Reply:
No problem :wink: